src/backend/distributed/replication/multi_logical_replication.c

```diff
@@ -349,6 +375,154 @@ LogicallyReplicateShards(List *shardList, char *sourceNodeName, int sourceNodePo
 }
 
 
+/*
+ * InitPublicationInfoHash initializes an empty hash map which can be used to
+ * store PublicationInfo entries.
+ */
+HTAB *
+InitPublicationInfoHash(void)
```

Comment:
minor, maybe: Init->Create

Reply:
Removed this method and used CreateSimpleHash directly.